### PR TITLE
Improve Core Information

### PR DESCRIPTION
**Pull request by lasers**
If merged, must merge https://github.com/libretro/libretro-super/pull/370 too.

### CODE REVIEW

**Comment by twinaphex** [comment]
Why are you hiding Core Notes?

**Comment by lasers** [comment]
Initial effort / experiment. I initially kept it in case somebody wanted to keep it. It's only later that I added "Remove all Core Notes" commit because it can be reverted back.

**Comment by twinaphex** [comment]
And why are you removing Core Notes?

**Comment by lasers** [comment]
I'm hiding/removing the label "Core Notes" that was sitting on its own line. The other labels have (Name: Value). This one does not... so it does not look clean / organized.

There is also firmware (checking, missing, required) which will use few spaces to pad information. It was barely noticeable and just looks slightly off. I switched it to (!). That's why I made that "Hide Core Notes" commit. I'm removing them because if you merge this in, then you wouldn't need "Core Notes".

```
(!) Information
[1] Notes
[*] Unorganized List
```

**Comment by twinaphex** [comment]
You'd have to ask @fr500 if he feels OK with removing that.

**Comment by lasers** [comment]
@fr500 Do you feel okay with this?

**Comment by fr500** [comment]
No... Why would you remove important info
Makes no sense whatsoever

**Comment by RobLoach** [comment]
This makes sense. All information on this page is showing notes about the core. Removing it cleans up the page.

## Before

corenotes2

## After

nocore

**Comment by fr500** [comment]
I don't really get... so it's only removing the header?

**Comment by RobLoach** [comment]
> so it's only removing the header?

Correct.

**Comment by fr500** [comment]
I don't like the obscure nomenclature he's using 
we should implement title and sublabels instead for this

File: menu/menu_displaylist.c [hunk 1]
```diff
@@ -381,15 +381,15 @@ static int menu_displaylist_parse_core_info(menu_displaylist_info_t *info)
          {
             if (core_info->firmware[i].desc)
             {
-               snprintf(tmp, sizeof(tmp), "	%s: %s",
+               snprintf(tmp, sizeof(tmp), "(!) %s: %s",
```
Comment (fr500) [inline menu/menu_displaylist.c]: you're adding this (!) here
and here https://github.com/libretro/libretro-super/pull/370/files

Won't this result in (!) (!)?

File: menu/menu_displaylist.c [hunk 2]
```diff
@@ -381,15 +381,15 @@ static int menu_displaylist_parse_core_info(menu_displaylist_info_t *info)
...
                      msg_hash_to_str(MENU_ENUM_LABEL_VALUE_RDB_ENTRY_NAME),
                      core_info->firmware[i].desc ?
                      core_info->firmware[i].desc : "");
                menu_entries_append_enum(info->list, tmp, "",
                      MENU_ENUM_LABEL_CORE_INFO_ENTRY,
                      MENU_SETTINGS_CORE_INFO_NONE, 0, 0);
 
-               snprintf(tmp, sizeof(tmp), "	%s: %s, %s",
+               snprintf(tmp, sizeof(tmp), "(!) %s: %s, %s",
```
Comment (RobLoach) [inline menu/menu_displaylist.c]: Another `(!)` reference.

**Comment by lasers** [comment]
@RobLoach, Get one from bsnes, please... and mame2014.

File: menu/menu_displaylist.c (hunk 1 above)
Comment (lasers) [inline menu/menu_displaylist.c]: This is for firmware_* only. I did notes manually.

**Pull request merged**